next/apps/Butler.py

```diff
@@ -142,17 +162,19 @@ def set(self, uid="", key=None, value=None, exp=None):
         """
         uid = (self.uid_prefix+uid).format(exp_uid=(self.exp_uid if exp is None else exp))
         if not key:
-            self.timed(self.db.set_doc)(self.collection, uid, value)
+            self.db.set_doc(self.collection, uid, value)
         else:
-            self.timed(self.db.set)(self.collection, uid, key, value)
+            self.db.set(self.collection, uid, key, value)
 
+    @timed
```

Comment:
I think you forgot an `op_type` on `set`, `set_many` and `append`. `timed` serves no purpose if `op_type` isn't set.